integration-cli/docker_cli_pause_test.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestEventsPause(t *testing.T) {
```

Comment:
Can you rename this to TestPause since its no longer part of the docker_cli_events_test.go file ?